src/schema/type/DefinedTerm.graphql

```diff
@@ -0,0 +1,70 @@
+"https://pending.schema.org/DefinedTerm"
+type DefinedTerm implements ThingInterface {
+    ### Metadata properties ###
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "http://purl.org/dc/elements/1.1/contributor,https://schema.org/contributor"
+    contributor: String
+    "http://purl.org/dc/elements/1.1/coverage"
+    coverage: String
+    "http://purl.org/dc/elements/1.1/creator"
+    creator: String!
+    "http://purl.org/dc/elements/1.1/date"
+    date: _Neo4jDate
+    "http://purl.org/dc/elements/1.1/description,https://schema.org/description"
+    description: String
+    "https://schema.org/disambiguatingDescription"
+    disambiguatingDescription: String
+    "http://purl.org/dc/elements/1.1/format"
+    format: String
+    "http://purl.org/dc/elements/1.1/language"
+    language: AvailableLanguage
+    "http://purl.org/dc/elements/1.1/publisher"
+    publisher: String
+    "http://purl.org/dc/elements/1.1/relation"
+    relation: String
+    "http://purl.org/dc/elements/1.1/rights"
+    rights: String
+    "http://purl.org/dc/elements/1.1/source"
+    source: String
+    "http://purl.org/dc/elements/1.1/subject"
+    subject: String
+    "http://purl.org/dc/elements/1.1/title"
+    title: String
+    "http://purl.org/dc/elements/1.1/type,https://www.w3.org/2000/01/rdf-schema#type"
+    type: String
+    "http://purl.org/dc/elements/1.1/date"
+    created: String
+    "http://purl.org/dc/elements/1.1/date"
+    updated: String
```

Comment:
there's no 'updated' field in dcterms, nor can I find one in schema.org. Is there any field defined for when an item was updated? If not, do we want to track if an item is updated? Does it make sense to track this?

Reply:
See #96

Reply:
https://github.com/trompamusic/ce-api/commit/9bf4d13203399440fe14559300f70802a8f9c6b4